### PR TITLE
Move templates and related code to OTE

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
ote-test/1386/

**Comment by Ilya-Krylov** [comment]
BMM 26833

**Comment by Ilya-Krylov** [comment]
ote-test 1389 ✔️

File: external/deep-object-reid/setup.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            if '-f http' in line:
+                links.append(line)
+            else:
+                requires.append(line)
+    return requires, links
+
+requirements, links = get_requirements('requirements.txt')
+
+print(requirements)
+print(links)
```
Comment (druzhkov-paul) [inline external/deep-object-reid/setup.py]: Don't forget to clean this up probably.

File: external/deep-object-reid/setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+setup(
+    name='torchreid_tasks',
```
Comment (druzhkov-paul) [inline external/deep-object-reid/setup.py:25]: @Ilya-Krylov  @sovrasov Is this the best name that we can come up with? I mean, that d-o-r is pretty much not torchreid at all, and torchreid_tasks is even harder to tie to what is under the hood. This being said, it's not a functional remark, so we can easily leave it as is.

File: external/deep-object-reid/init_venv.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  echo "Using CUDA_VERSION ${CUDA_VERSION}"
+  if [[ "${CUDA_VERSION_CODE}" != "111" ]] && [[ "${CUDA_VERSION_CODE}" != "102" ]] ; then
+    echo "CUDA version must be either 11.1 or 10.2"
+    exit 1
+  fi
+  echo "export CUDA_HOME=${CUDA_HOME}" >> ${venv_dir}/bin/activate
+fi
+
+CONSTRAINTS_FILE=$(tempfile)
+cat constraints.txt >> ${CONSTRAINTS_FILE}
```
Comment (druzhkov-paul) [inline external/deep-object-reid/init_venv.sh:85]: Is it intentional that constraints.txt is blank?

File: external/deep-object-reid/tools/ote_sample.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (druzhkov-paul) [inline external/deep-object-reid/tools/ote_sample.py:1]: Does it make any sense to still keep sample when it's located side by side with CLI?

File: external/deep-object-reid/setup.py (hunk 1 above)
Comment (Ilya-Krylov) [reply external/deep-object-reid/setup.py]: thx

File: external/deep-object-reid/tools/ote_sample.py (hunk 4 above)
Comment (Ilya-Krylov) [reply external/deep-object-reid/tools/ote_sample.py:1]: probably not, @sovrasov ?

File: external/deep-object-reid/setup.py (hunk 2 above)
Comment (Ilya-Krylov) [reply external/deep-object-reid/setup.py:25]: will deep_object_reid_tasks be better?

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: external/deep-object-reid/init_venv.sh (hunk 3 above)
Comment (Ilya-Krylov) [reply external/deep-object-reid/init_venv.sh:85]: will fill it later

File: external/deep-object-reid/tools/ote_sample.py (hunk 4 above)
Comment (Ilya-Krylov) [reply external/deep-object-reid/tools/ote_sample.py:1]: @sovrasov please respond, will remove it later if it is needed

File: external/deep-object-reid/setup.py (hunk 2 above)
Comment (sovrasov) [reply external/deep-object-reid/setup.py:25]: My proposal is `classification_tasks`. deep_object_reid also doesn't make sense.

File: external/deep-object-reid/tools/ote_sample.py (hunk 4 above)
Comment (sovrasov) [reply external/deep-object-reid/tools/ote_sample.py:1]: Yes, we can remove it

**Pull request merged**